com.woltlab.wcf/templates/shared_bbcode_wsm.tpl

```diff
@@ -4,7 +4,7 @@
 	{if $media->isImage}
 		{if $thumbnailSize != 'original'}
 			{if !$removeLinks}
-				<a href="{$mediaLink}" class="embeddedAttachmentLink jsImageViewer">
+				<a href="{$mediaLink}" data-caption="{$media->title}" class="embeddedAttachmentLink" data-fancybox="attachments">
```

Comment:
Why is this `data-fancybox="attachments"`? The BBCode can appear in other places that have nothing to do with attachments.